app/assets/javascripts/admin/addon/components/admin-config-area-cards/about/contact-information.gjs

```diff
@@ -1,20 +1,82 @@
 import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { hash } from "@ember/helper";
+import { on } from "@ember/modifier";
 import { action } from "@ember/object";
+import { service } from "@ember/service";
 import DButton from "discourse/components/d-button";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
 import i18n from "discourse-common/helpers/i18n";
 import GroupChooser from "select-kit/components/group-chooser";
 import UserChooser from "select-kit/components/user-chooser";
 
 export default class AdminConfigAreasAboutContactInformation extends Component {
+  @service site;
+
+  @tracked showSavedAlert = false;
+  @tracked
+  contactUsername = this.args.contactInformation.contactUsername.value || null;
+  @tracked
+  contactGroupId = this.site.groups.find(
+    (group) => group.name === this.contactGroupName
+  )?.id;
+
+  communityOwner = this.args.contactInformation.communityOwner.value;
+  contactEmail = this.args.contactInformation.contactEmail.value;
+  contactURL = this.args.contactInformation.contactURL.value;
+  contactGroupName = this.args.contactInformation.contactGroupName.value;
+
+  @action
+  onCommunityOwnerChange(event) {
+    this.communityOwner = event.target.value;
+  }
+
+  @action
+  onContactEmailChange(event) {
+    this.contactEmail = event.target.value;
+  }
+
+  @action
+  onContactURLChange(event) {
+    this.contactURL = event.target.value;
+  }
+
+  @action
+  onContactUsernameChange(usernames) {
+    this.contactUsername = usernames[0];
+  }
+
+  @action
+  onContactGroupIdChange(ids, groups) {
+    this.contactGroupId = ids[0];
+    this.contactGroupName = groups[0]?.name;
+  }
+
   @action
-  save() {
-    this.args.saveCallback();
-    // eslint-disable-next-line no-console
-    console.log("contact information");
+  async save() {
+    this.showSavedAlert = false;
+    try {
+      await ajax("/admin/config/about.json", {
+        type: "PUT",
+        data: {
+          contact_information: {
+            community_owner: this.communityOwner,
+            contact_email: this.contactEmail,
+            contact_url: this.contactURL,
+            contact_username: this.contactUsername,
+            contact_group_name: this.contactGroupName,
+          },
+        },
+      });
+      this.showSavedAlert = true;
```

Comment:
Should this be done in a `finally` in case anything bad happens during the `ajax` request?

Reply:
Yes I agree with Regis here.

Reply:
No, the "saved!" indicator should only show up when the ajax calls succeeds so we don't want to set `showSavedAlert` in a `finally` block.